### DISCUSSION TITLE
Search Query Tracking

### DISCUSSION THREAD

**Discussion by Lucas-SZ** [requarks/wiki#3747] [General]
Hello Everyone,

In Wiki.JS 2.5 has anyone had luck tracking search queries using google analytics or another service?  There does not appear to be a standard /q /search parameter used.

Thanks

**Comment by phewitt-uw** [comment]
I am looking for the exact same thing! For me it looked like using Algolia as a search engine provider would do the trick but sadly the WIKIJS implementation does not account for Algolia's upload limits to the service (pages <100KB)

I does raise a bit of a concern they are happy to advertise Algolia as a working search module which this is plain wrong considering it does not work. I have seen they have put the blame on Algolia due to the size restrictions but to be honest, if you aren't going to adhere to their service requirements then you shouldn't offer the module as a working option!

Thats my 2 cents anyways...
